test/libsolidity/syntaxTests/types/function_types/selector/local_variable_selector_not_pure.sol

```diff
@@ -0,0 +1,6 @@
+contract C {
+    function f() public pure returns (bytes4) {
+        function() external g;
+        g.selector;
```

Comment:
Note the absence of a ``Statement has no effect`` warning.

Reply:
Can you add a comment?

Reply:
Added a comment.